test/e2e_env/multizone/trafficpermission/trafficpermission.go

```diff
@@ -0,0 +1,178 @@
+package trafficpermission
+
+import (
+	"fmt"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	core_mesh "github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
+	"github.com/kumahq/kuma/test/e2e_env/multizone/env"
+	. "github.com/kumahq/kuma/test/framework"
+)
+
+func TrafficPermission() {
+	const meshName = "tp-test"
+	const namespace = "tp-test"
```

Comment:
Can we autogenerate this using UUID suffix for example?

Reply:
I'd prefer to keep the name stable. It's easier to work with a test if you don't have to extract the mesh/namespace main from the test with every run.

Reply:
Maybe we can construct it somehow using `ginkgo.CurrentGinkgoTestDescription()`:
```go
//GinkgoTestDescription represents the information about the current running test returned by CurrentGinkgoTestDescription
//	FullTestText: a concatenation of ComponentTexts and the TestText
//	ComponentTexts: a list of all texts for the Describes & Contexts leading up to the current test
//	TestText: the text in the actual It or Measure node
//	IsMeasurement: true if the current test is a measurement
//	FileName: the name of the file containing the current test
//	LineNumber: the line number for the current test
//	Failed: if the current test has failed, this will be true (useful in an AfterEach)
type GinkgoTestDescription struct {
	FullTestText   string
	ComponentTexts []string
	TestText       string

	IsMeasurement bool

	FileName   string
	LineNumber int

	Failed   bool
	Duration time.Duration
}
```
but it's not a blocker for this PR

Reply:
it is an option, but not sure about length and spaces, etc. I'd keep this static and do autogeneration if we ever found this problematic